system-tests/projects/runner-e2e-specs/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
I think we should consider against adding this here - this will mean `yarn install` runs on every project using `runner-e2e-specs`, which is a LOT.

Also this is runner-e2e-specs - maybe we want `runner-ct-specs`?

Reply:
Also every project using runner-e2e-specs seems to be broken on windows CI - finding out why...